### DISCUSSION TITLE
Re-build specific resource versions?

### DISCUSSION THREAD

**Discussion by patmaddox** [concourse/concourse#7918] [Help & Support] [answered]
I restarted our worker, which seems to have cleared the resource versions cache. When I try to "re-run with the same inputs" it fails because the resource version isn't found - even though the ref specified by the version is in the git history.

Is there any way to re-build that specific resource version so that I can "re-run with the same inputs"?

**Comment by taylorsilva** [comment, marked as answer]
Did your resource lose it's version history as well? Pressing the re-run button should work unless the version history got cleared (I think).
You can have the resource pull more versions/commits in by doing `fly check-resource ... --from "ref:some-older-commit-sha"` and it should pull in all commits after the provided commit.

**Reply by patmaddox** [reply]
> You can have the resource pull more versions/commits in by doing `fly check-resource ... --from "ref:some-older-commit-sha"` and it should pull in all commits after the provided commit.

That's it! Thanks :)
